var/spack/repos/builtin/packages/reprimand/package.py

```diff
@@ -19,6 +21,7 @@ class Reprimand(MesonPackage):
     maintainers = ["eschnett"]
 
     version("develop", git="https://github.com/wokast/RePrimAnd", branch="public")
+    version("1.5", sha256="bc71030c1ae337c3631cfc7e46270260b0663e4ad73129148bf443c9220afb86")
     version("1.4", sha256="260730696175fa21d35d1a92df2c68b69243bb617083c82616efcb4720d557e8")
     version("1.3", sha256="8e9f05b1f065a876d1405562285a9f64d1b31c4a436d5a6bb1f023212b40314e")
```

Comment:
I'm not getting these sha256:

```
$ spack checksum reprimand 1.5 1.4 1.3
==> Found 3 versions of reprimand:
  
  1.5  https://github.com/wokast/RePrimAnd/archive/refs/tags/v1.5.tar.gz
  1.4  https://github.com/wokast/RePrimAnd/archive/refs/tags/v1.4.tar.gz
  1.3  https://github.com/wokast/RePrimAnd/archive/refs/tags/v1.3.tar.gz

==> Fetching https://github.com/wokast/RePrimAnd/archive/refs/tags/v1.5.tar.gz
==> Fetching https://github.com/wokast/RePrimAnd/archive/refs/tags/v1.4.tar.gz
==> Fetching https://github.com/wokast/RePrimAnd/archive/refs/tags/v1.3.tar.gz

    version("1.5", sha256="0f1d65a170cad7bc071c58b747b2f0bdc1df77e4c56c152dfb6c93a9d1360f28")
    version("1.4", sha256="7ddeb770b5b2e2826c5889a38427fa201bb6b9ff98b4ded291ea730ad0e61a42")
    version("1.3", sha256="2b9d016f0d2d3d11082f049c2b80e57a8c3ea4d5e3875a863c7c485767475ee5")
```

Reply:
Weird. I just ran `spack fetch`, and I installed the package, and this worked fine with the checksums above.

Reply:
I can `spack fetch` and `spack checksum` the package with both checksums (yours and mine)! What sorcery is this?

Reply:
Just so I'm clear, are you saying you're getting the same `spack checksum` results I posted above?  Or are you saying those in this PR are correct?

Reply:
Yes, I am getting the same results as you. I will update the pull request accordingly.

Reply:
Done.